controllers/story-controller.ts

```diff
@@ -137,8 +137,16 @@ export default class StoryController implements StoryControllerI {
    * @param {Response} res Represents response to client, including the
    * body formatted as JSON arrays containing the story objects
    */
-  findStories = (req: Request, res: Response) => {
-      StoryController.storyDao.findStories().then((stories: Story[]) => res.json(stories));
+  findStories = async (req: Request, res: Response) => {
+    const stories = await StoryController.storyDao.findStories();
+    const twentyFourHours = new Date().getTime() + (24 * 60 * 60 * 1000);
```

Comment:
We should be accepting hours as a parameter
`
if (req.query.hours) {
    const hours = new Date().getTime() - (req.query.hours * 60 * 60 * 1000);
    const filteredStories = stories.filter((story) => hours <= story.postedOn.getTime());
      return res.json(filteredStories);
    }
`

Reply:
Fixed!